### PR TITLE
feat(controls): Add react versions of existing media control icons

### PR DESCRIPTION
**Pull request by jstoffan**

### CODE REVIEW

File: src/lib/viewers/controls/icons/IconVolumeLow24.tsx [hunk 1]
```diff
@@ -0,0 +1,12 @@
+import * as React from 'react';
+
+function IconVolumeLow24(props: React.SVGProps<SVGSVGElement>): JSX.Element {
+    return (
+        <svg focusable={false} height={24} viewBox="-3 5 24 24" width={24} {...props}>
+            <path d="M0 14v6h4l5 5V9l-5 5H0z" fill="#FFF" />
```
Comment (ConradJChan) [inline src/lib/viewers/controls/icons/IconVolumeLow24.tsx:6]: Should these `#fff` and `#FFF` be consistent?

File: src/lib/viewers/controls/icons/IconVolumeLow24.tsx [hunk 2]
```diff
@@ -0,0 +1,12 @@
+import * as React from 'react';
+
+function IconVolumeLow24(props: React.SVGProps<SVGSVGElement>): JSX.Element {
+    return (
+        <svg focusable={false} height={24} viewBox="-3 5 24 24" width={24} {...props}>
```
Comment (ConradJChan) [inline src/lib/viewers/controls/icons/IconVolumeLow24.tsx:5]: Are these negative values in the viewBox worrisome?

**Review by ConradJChan — APPROVED** [review]

File: src/lib/viewers/controls/icons/IconVolumeLow24.tsx (hunk 2 above)
Comment (jstoffan) [reply src/lib/viewers/controls/icons/IconVolumeLow24.tsx:5]: The SVGs are copied directly from the existing icons. I'd rather not mess with them much.

File: src/lib/viewers/controls/icons/IconVolumeLow24.tsx (hunk 1 above)
Comment (jstoffan) [reply src/lib/viewers/controls/icons/IconVolumeLow24.tsx:6]: The other icons are already inconsistent. I can do this separately if it's important?

Comment (cweeii) [reply src/lib/viewers/controls/icons/IconVolumeLow24.tsx:6]: Should we start using the buie `$white` color?

**Pull request merged**